lib/src/test/java/ca/jbrains/pos/test/TestSellOneItem.java

```diff
@@ -9,32 +12,38 @@
     void priceNotFound() {
         Assertions.assertEquals(
                 "Product not found: 99999",
-                PointOfSale.displaySellOneItem("99999", (ignored) -> null)
+                PointOfSale.displaySellOneItem(barcode -> Either.left(() -> "99999"), Barcode.parse("99999"))
         );
     }
 
     @Test
     void givenBarcodeIs1111ShouldDisplayProductNotFoundMessage() {
         Assertions.assertEquals(
                 "Product not found: 1111",
-                PointOfSale.displaySellOneItem("1111", (ignored) -> null)
+                PointOfSale.displaySellOneItem(barcode -> Either.left(() -> "1111"), Barcode.parse("1111"))
         );
     }
 
     @Test
     void priceFound() {
         Assertions.assertEquals(
                 "CAD 1.00",
-                PointOfSale.displaySellOneItem("99999", (ignored) -> "CAD 1.00")
+                PointOfSale.displaySellOneItem(ignored -> Either.right(new StubProductFound()), Barcode.parse("99999"))
         );
     }
 
     @Test
-    void givenEmptyBarcodeShouldReturnScanningErrorMessage() {
-        Assertions.assertEquals(
-                "Scanning error: empty barcode",
-                PointOfSale.displaySellOneItem("", (ignored) -> null)
-        );
+    void shouldFormat101Correctly() {
+        Assertions.assertEquals("CAD 1.01", PointOfSale.formatPrice(new Price(101)));
     }
 
+    @Test
+    void shouldFormat110Correctly() {
```

Comment:
What is interesting or significant about the value `110` that makes you want to check it? What about `100`? Try renaming the tests to be the answers to those questions.

Reply:
right, done